x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/store/index.test.ts

```diff
@@ -43,7 +43,9 @@ describe('EndpointList store concerns', () => {
         error: undefined,
         endpointDetails: {
           activityLog: {
-            type: 'UninitialisedResourceState',
+            page: 1,
+            pageSize: 10,
+            logData: { type: 'UninitialisedResourceState' },
```

Comment:
createUninitialisedResourceState()

Reply:
For some reason, the rest of the code block does it this way. I would guess it is so that the test is more obvious and readable. See https://github.com/elastic/kibana/blob/b740640a2ad5f69d732c177cea0fa96bc424c257/x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/store/index.test.ts#L76